Dockerfile.frontend.local

```diff
@@ -0,0 +1,29 @@
+ARG METADATASERVICE_BASE
+ARG SEARCHSERVICE_BASE
+
+FROM node:12-slim as node-stage
+WORKDIR /app/amundsen_application/static
+
+COPY ./frontend/amundsen_application/static/package.json /app/amundsen_application/static/package.json
+COPY ./frontend/amundsen_application/static/package-lock.json /app/amundsen_application/static/package-lock.json
+RUN npm install
+
+COPY ./frontend/amundsen_application/static/ /app/amundsen_application/static/
+RUN npm rebuild node-sass
+RUN npm run dev-build
+
+COPY ./frontend /app
+
+FROM python:3.7-slim
+WORKDIR /app
+
+COPY ./frontend /app
+COPY requirements-dev.txt /app/requirements-dev.txt
```

Comment:
Are there dev requirements needed for the container?

Reply:
without this file `pip install -e .[all]` fails cause `requirements-dev.txt` is missing (as `setup.py` fails on missing file when reading in to `requirements_dev` variable). 
I can either:
- remove this file from `Dockerfile` and introduce handling of missing files in `setup.py`  (load empty array to `requirements_dev` variable if the file is missing)
- keep this file in `Dockerfile` (they are not installed actually, just the file needs to be there `setup.py` to execute correctly) and not amend `setup.py` 

my preference is the latter one but if you feel differently I can make the change.

Reply:
Thanks for the context. I agree with this approach. This PR was also about unifying the requirements, which appears to have been done. Should this be an issue in the future it can be addressed then.

Reply:
I've updated README.md for search and metadata + `docs/installation.md` for frontend. Should I add it also somewhere else?